task/rsync/0.1/rsync.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rsync
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
```

Comment:
also since optional workspaces are being used over here can you please change the `pipelines.minVersion` to `0.17.x`

Reply:
fixed.